frontend/src/layout/navigation-3000/components/ProjectTree/projectTreeLogic.tsx

```diff
@@ -40,6 +40,12 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
         createSavedItem: (savedItem: FileSystemEntry) => ({ savedItem }),
         updateSavedItem: (savedItem: FileSystemEntry) => ({ savedItem }),
         deleteSavedItem: (savedItem: FileSystemEntry) => ({ savedItem }),
+        updateExpandedFolders: (folders: string[]) => ({ folders }),
+        updateActiveFolder: (folder: string | null) => ({ folder }),
+        updateLastViewedPath: (path: string) => ({ path }),
+        toggleFolder: (folder: string, isExpanded: boolean) => ({ folder, isExpanded }),
+        updateSelectedFolder: (folder: string) => ({ folder }),
+        updateHelpNoticeVisibility: (visible: boolean) => ({ visible }),
```

Comment:
I don't know if everyone follows this pattern, but I've usually seen `update` as an action that merges data, and `set` as something that replaces it. Here all these actions just set the fields directly, so I'd rename them all to `setSelectedFolder`, etc.

I remarked earlier that the "action setX, reducer X" pattern in Kea is a bit of an anti-pattern... but sometimes you can't avoid it. In those cases, might as well call a spade a spade, and a set a set.